src/Debug/Gui.cpp

```diff
@@ -452,27 +452,28 @@ bool Gui::ShowMenu(Game& game)
 				}
 			});
 
-			auto menuItem = [&game](const auto& label, const std::filesystem::path& path) {
-				if (ImGui::MenuItem(label.data()))
+			auto menuItem = [&game](const auto& label, const std::filesystem::path& path, const std::string description,
+			                        bool validLevel) {
+				if (ImGui::MenuItem(label.data(), nullptr, false, validLevel))
 				{
 					game.LoadMap(path);
 				}
 				if (ImGui::IsItemHovered())
 				{
-					ImGui::SetTooltip("%s", path.generic_string().c_str());
+					ImGui::SetTooltip("%s", description.c_str());
 				}
 			};
 
 			ImGui::MenuItem("Story Islands", nullptr, false, false);
 			for (auto& level : campaigns)
 			{
-				menuItem(level->GetName(), level->GetScriptPath());
+				menuItem(level->GetName(), level->GetScriptPath(), level->GetScriptPath().generic_string(), level->IsValid());
```

Comment:
Is this on purpose to have the path instead of the description?
Is the `IsValid` call making the item disabled?

Reply:
> Is this on purpose to have the path instead of the description?
It was on purpose but it was misktaken, i used the path as the description are extracted from ADD_GAME_MESSAGE_LINE inside script file and campaign map don't have this instruction. Now the loader build the description and you're just supposed to get it when needed.

> Is the IsValid call making the item disabled?
Yes, if i can't find LOAD_LANDSCAPE instruction the item is disabled.